pkgs/development/python-modules/transitions/default.nix

```diff
@@ -50,6 +50,9 @@ buildPythonPackage rec {
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
       # Upstream issue https://github.com/pygraphviz/pygraphviz/issues/441
       "test_binary_stream"
+
+      # sleep is not accurate on Darwin
+      "test_timeout_callbacks"
```

Comment:
```suggestion
      "test_timeout_callbacks"
      "test_timeout_transitioning"
```

Reply:
Interesting. I can't reproduce this bug; it’s likely machine-specific. To avoid it, I’ve added sleep to all relevant tests.

Reply:
Yes, it's checking execution time against a set value and fails if the machine is running slowly. It appeared in the x86-darwin Hydra build since those boxes are *slammed*.